### PR TITLE
created html framework initial stages

### PR DESCRIPTION
**Pull request by sashajustice**

### CODE REVIEW

File: calculator.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Simple Calculator</title>
+</head>
+<body>
+    <div class="display"><input type="text" size="15.75" id="display" read only>
```
Comment (shakalee14) [inline calculator.html]: i think the readonly attribute should be one word 

```<input type="text" size="15.75" id="display" readonly> ```

File: calculator.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <div class="keys">
+      <p>
+          <input type="button" class="button" value="(" onclick='math'("(")'>
```
Comment (shakalee14) [inline calculator.html]: are you going to define math() in another file?

File: calculator.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          <input type="button" class="button" value="8" onclick='math("")'>
+          <input type="button" class="button" value="9" onclick='math("")'>
+          <input type="button" class="button" value="*" onclick='math("")'>
+      </p>
+      <p>
+          <input type="button" class="button" value="4" onclick='math("")'>
+          <input type="button" class="button" value="5" onclick='math("")'>
+          <input type="button" class="button" value="6" onclick='math("")'>
+          <input type="button" class="button" value="-" onclick='math("")'>
+    </p>
```
Comment (shakalee14) [inline calculator.html]: i think to match your other tabbing, </p> on 26 and <p> should be tabbed in one more, or everything else should be tabbed out.

File: calculator.html (hunk 1 above)
Comment (sashajustice) [inline calculator.html]: Oh, ok thanks

File: calculator.html (hunk 3 above)
Comment (sashajustice) [inline calculator.html]: Ok.

File: calculator.html (hunk 2 above)
Comment (sashajustice) [inline calculator.html]: Not sure yet..

**Pull request merged**